synapse/api/constants.py

```diff
@@ -27,16 +27,6 @@ class Membership(object):
     LIST = (INVITE, JOIN, KNOCK, LEAVE, BAN)
 
 
-class Feedback(object):
```

Comment:
Yeah. It didn't seem worth breaking out of flow to setup a separate PR at the time.